cfgov/unprocessed/js/molecules/Expandable.js

```diff
@@ -0,0 +1,343 @@
+'use strict';
+
+// Required polyfills for <IE9.
+require( '../modules/polyfill/query-selector' );
+require( '../modules/polyfill/event-listener' );
+require( '../modules/polyfill/class-list' );
+
+// Required modules.
+var EventObserver = require( '../modules/util/EventObserver' );
+
+/**
+ * Expandable
+ * @class
+ *
+ * @classdesc Initializes a new Expandable molecule.
+ *
+ * @param {HTMLNode} element
+ *   The DOM element within which to search for the molecule.
+ * @returns {Object} An Expandable instance.
+ */
+function Expandable( element ) { // eslint-disable-line max-statements, inline-comments, max-len
+
+  var BASE_CLASS = 'm-expandable';
+
+  // Bitwise flags for the state of this Expandable.
+  var COLLAPSED = 0;
+  var COLLAPSING = 1;
+  var EXPANDING = 2;
+  var EXPANDED = 3;
+
+  // The Expandable element will directly be the Expandable
+  // when used in an ExpandableGroup, otherwise it can be the parent container.
+  var _dom = element.classList.contains( BASE_CLASS ) ?
+             element : element.querySelector( '.' + BASE_CLASS );
+  var _target = _dom.querySelector( '.' + BASE_CLASS + '_target' );
+  var _content = _dom.querySelector( '.' + BASE_CLASS + '_content' );
+  var _contentAnimated =
+    _content.querySelector( '.' + BASE_CLASS + '_content-animated' );
+  var _link = _dom.querySelector( '.' + BASE_CLASS + '_link' );
+
+  var _state = 0;
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/2d403e1899097e831033443b174af07d2a389545